submissions/Islamic_II/Persian/get_persian.xquery

```diff
@@ -0,0 +1,13 @@
+xquery version "3.0";
+declare option saxon:output "omit-xml-declaration=yes";
+
+declare variable $marc as document-node()+ := doc("file:/Users/heberleinr/Documents/Digital_Scriptorium/submissions/2024_08/Islamic_II/BIBLIOGRAPHIC_38955194630006421_1.xml");
```

Comment:
Do you want this to be an absolute path to a file on your computer? Is there a way to make it more generalized?